docs/test/live-unit-testing-faq.md

```diff
@@ -51,6 +51,16 @@ Live Unit testing may not work for one of the following reasons:
     </RunSettings> 
    ``` 
 
+## Why does Live Unit Testing show incorrect coverage after upgrading referenced Visual Studio test adapter in projects to supported version?
```

Comment:
This is phrased awkwardly. Better: "Why does Live Unit Testing show incorrect coverage after you upgrade the test adapter referenced in your Visual Studio Projects to the supported version?"